astropy/io/fits/convenience.py

```diff
@@ -409,8 +409,16 @@ def writeto(filename, data, header=None, output_verify='exception',
         for more info.
 
     clobber : bool, optional
-        If `True`, and if filename already exists, it will overwrite
-        the file.  Default is `False`.
+        If `True`, overwrite the output file if exists.
+        The use of clobber is deprecated; use overwrite instead.
+
+    overwrite : bool, optional
```

Comment:
Sphinx has a [`versionadded`](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded) and [`deprecated`](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-deprecated)-directive. I haven't seen it regularly used but I think it could make sense here.

For `clobber` this would be something like 

```
.. deprecated:: 1.3
   Use the ``overwrite`` argument instead.
```

and for overwrite: `.. versionadded:: 1.3`